packages/types/src/entities/index.ts

```diff
@@ -6,6 +6,8 @@ export type * from './nav.js';
 export type FileDescriptor = { path: string; type: 'file' | 'folder' };
 export type Files = Record<string, string | Uint8Array>;
 
+export type FilesystemError = 'FILE_EXISTS' | 'FOLDER_EXISTS';
```

Comment:
I thought about using real `Error` here, like `class FilesystemError extends Error {}` but maybe we can use something as simple as string literals here? Any thoughts?

Reply:
I think this is fine, we could also use an enum but a string literal is fine by me :+1: